src/interactivity/win32/windowio.cpp

```diff
@@ -635,6 +638,20 @@ BOOL HandleMouseEvent(const SCREEN_INFORMATION& ScreenInfo,
 
         if (HandleTerminalMouseEvent(MousePosition, Message, GET_KEYSTATE_WPARAM(wParam), sDelta))
         {
+            switch (Message)

```

Comment:
I'd be inclined also to leave a quick comment here about why capturing is great in VT mode like what sorts of applications it enables or what broken behaviors result if we don't do it. (And maybe that it only happens in VT mode BECAUSE the `HandleTerminalMouseEvent` call should return `false` outside VT mode... codifying the assumption here.)